test/gurka/test_pbf_api.cc

```diff
@@ -91,6 +105,10 @@ TEST(pbf_api, pbf_error) {
       auto pbf_bytes = serialize_error(e, api);
       Api actual;
       EXPECT_TRUE(actual.ParseFromString(pbf_bytes));
+      EXPECT_FALSE(actual.has_options());
```

Comment:
here i just make sure that you only get back the info object with the error message and nothign else when an error occurs